collectors/osd.go

```diff
@@ -68,6 +91,10 @@ type OSDCollector struct {
 
 	// AverageUtil displays average utilization in all OSDs
 	AverageUtil prometheus.Gauge
+
+	// ScrubbingState depicts the state of scrub on a given osd
+	// labelled by OSD
+	ScrubbingState *prometheus.GaugeVec
```

Comment:
You can use [`MustNewConstMetric()`](https://godoc.org/github.com/prometheus/client_golang/prometheus#MustNewConstMetric) instead here since we need zero values when the values don't appear in the next run.

An example use is here: https://github.com/digitalocean/ceph_exporter/blob/luminous/collectors/health.go#L931.